aea/registries/base.py

```diff
@@ -106,7 +114,7 @@ def teardown(self) -> None:
         """
 
 
-class ProtocolRegistry(Registry):
+class ProtocolRegistry(Registry[PublicId, Protocol]):
```

Comment:
a `ProtocolRegistry` implements the `Registry` interface. The `ItemId` in this case is a `PublicId` (to be changed to `ProtocolId`...) and the `Item` type is `Protocol`.

Reply:
Cool!